<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTSPageSysConfig</class>
 <widget class="QWidget" name="DTSPageSysConfig">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>647</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>647</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>647</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>系统参数配置</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>341</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="14" column="0">
       <widget class="QPushButton" name="m_pbTestDB">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>测试数据库</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2" rowspan="5">
       <widget class="QLabel" name="m_lblCheckDB">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="m_leDBPass">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="m_lblDBPass">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>登录密码：</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="m_leDBPort">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="m_leDBUser">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="QLabel" name="m_lblCheckWS">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="m_lblDBName">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>默认名称：</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <spacer name="verticalSpacer3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="m_leDBName">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="m_leDBAddr">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QLabel" name="m_lblCheckApp">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <spacer name="verticalSpacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_leAppID">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_leAppName">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="m_lblWSAddr">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>监听地址：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="m_lblAppID">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>应用编号：</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="m_lblDBUser">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>登录用户：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_lblAppName">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>应用名称：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="m_leWSPort">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLabel" name="m_lblDB">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>本地 数据库</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="m_leWSAddr">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="m_lblWSPort">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>监听端口：</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="m_lblDBAddr">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>监听地址：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="m_lblWS">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>本地 WebService服务</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="m_lblDBPort">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>监听端口：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="m_pbTestWS">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>测试服务</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="m_lblTipsWS">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: &quot;red&quot;</string>
        </property>
        <property name="text">
         <string>重启生效！</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="m_pbUseDef">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>恢复默认</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_pbCancel">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pbCreate">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
